dom/src/index.ts

```diff
@@ -29,10 +29,12 @@ export {MainDOMSource} from './MainDOMSource';
  * The other option is `preventDefault` that is set to false by default.
  * If set to true, the driver will automatically call `preventDefault()` on every event.
  *
- * `DOMSource.elements()` returns a stream of the DOM element(s) matched by the
- * selectors in the DOMSource. Also, `DOMSource.select(':root').elements()`
- * returns a stream of DOM element corresponding to the root (or container) of
- * the app on the DOM.
+ * `DOMSource.elements()` returns a stream of arrays of the DOM elements matched by the selectors in the DOMSource. Also, `DOMSource.select(':root').elements()`
```

Comment:
Probably nitpick, but since you'll update the commit, could keep this line less than 80 columns.